File: MVVM/Converters/ModuloConverter.vb
Imports System.Globalization
Imports System.Windows.Data
Imports System.Windows.Media

Namespace MVVM.Converters

    Public Class ModuloConverter
        Implements IValueConverter

        Public Function Convert(ByVal Value As Object, ByVal TargetType As Type, ByVal Parameter As Object, ByVal Culture As CultureInfo) As Object Implements IValueConverter.Convert
            Dim A = CType(Value, Integer)
            Dim B = CType(Parameter, Integer)

            Return A Mod B
        End Function

        Public Function ConvertBack(ByVal Value As Object, ByVal TargetType As Type, ByVal Parameter As Object, ByVal Culture As CultureInfo) As Object Implements IValueConverter.ConvertBack
            Return Value
        End Function

    End Class

End Namespace

File: WPF/MVVM/Converters/ModuloConverter.cs
using System;
using System.Globalization;
using System.Windows.Data;

namespace Ks
{
    namespace Common.MVVM.Converters
    {
        public class ModuloConverter : IValueConverter
        {
            public object Convert(object Value, Type TargetType, object Parameter, CultureInfo Culture)
            {
                var A = System.Convert.ToInt32(Value);
                var B = System.Convert.ToInt32(Parameter);

                return A % B;
            }

            public object ConvertBack(object Value, Type TargetType, object Parameter, CultureInfo Culture)
            {
                return Value;
            }
        }
    }
}
